systemd/units/init.sls

```diff
@@ -7,10 +7,16 @@ include:
   {% if unittype in unittypes.get('Valid') %}
     {% for unit, unitconfig in units.items() %}
       {% set unit_status = 'disable' if unitconfig.enabled is defined and unitconfig.enabled == false else 'enable' %}
+      {% set dropin = true if unitconfig.dropin is defined and unitconfig.dropin == true else false %}
 
```

Comment:
@toanju this LGTM, but just a suggestion here:

I think I'd prefer to set these two as
```jinja
    {% set unit_status = unitconfig.enabled | default ('enable') %}
    {% set dropin = unitconfig.dropin | default(false) %}
```

to make the code clearer? wdyt?

Reply:
I was keeping the original format. However, I agree with you that this makes the code look cleaner. Let me update the PR and refactor the other cases.

Reply:
looking a bit closer at this, I guess this will work only for the `dropin` part but not for the `unit_status` as we need either `disable` or `enable` as a result having the input `true/false`. However, do you see a better option for `unit_status`?

Reply:
You're right, I missed that.